db/data/scraped_optional_subjects.yml

```diff
@@ -0,0 +1,42 @@
+---
+- '5914'
```

Comment:
probably no need to make them strings (ie: `- 5914`)

Reply:
Now the scrapper generates this file